src/main/java/com/custom/fuelsdk/ETNotSentEvent.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.custom.fuelsdk;
+
+import com.exacttarget.fuelsdk.ETSoapObject;
+import com.exacttarget.fuelsdk.annotations.ExternalName;
+import com.exacttarget.fuelsdk.annotations.InternalName;
+import com.exacttarget.fuelsdk.annotations.SoapObject;
+import com.exacttarget.fuelsdk.internal.EventType;
+import com.exacttarget.fuelsdk.internal.NotSentEvent;
+
+import java.util.Date;
+
+/**
+ * An <code>ETNotSentEvent</code> object represents information pertaining to the notsent events
+ * of an email message in the Salesforce Marketing Cloud.
+ * This object type is currently not supported in FuelSDK and hence the provided here.
+ */
+@SoapObject(internalType = NotSentEvent.class)
+public class ETNotSentEvent extends ETSoapObject {
```

Comment:
where is this used?

Reply:
This is one of the tracking events that's supported by the SFMC SOAP API but was not implemented in fuelSDK java api.  The fix contains the implementation for this event type.